tests/test_imports.py

```diff
@@ -40,8 +40,8 @@ def test_activity_mapper(gold_mapper):
     gold_mapper.activity_mapper()
     gold_mapper.post_nmdc_database_object()
     db = gold_mapper.get_database_object_dump()
-    assert len(db.metagenome_annotation_activity_set) == 1
-    assert len(db.mags_activity_set) == 1
+    #TODO I think this assert statement should be 2 ANC
```

Comment:
@mbthornton-lbl please double check correct value for len(db.workflow_execution_set) assertion